clouddq/log.py

```diff
@@ -64,14 +64,15 @@ def format(self, record):
         return super().format(record)
 
 
-def add_cloud_logging_handler(logger: Logger):
+def add_cloud_logging_handler(metadata, logger: Logger):
```

Comment:
please add type hint, also new arguments should be added *after* existing arguments, rather than before